PayButton/paybutton-server
# Feat/user timezone

Related to #893 






Description
---
Implement timezone option in user Account settings. 
This PR will enable timezone option, the selector  is initialized with the timezone of the user by location, once user changes the selector value it will set the timezone and it will make the dashboard data, the txs listing of each button and payments listing to use the timezone selected. 

Test plan
---
Start the server with `docker compose up`
- Go to account page, check if selector is initialized with timezone of your location
- Change selector to another timezone
- Check payments list see if the date of each payment is in the preferred timezone  
- Check the transaction list in button page see if the date of each tx is in the preferred timezone 
- Check dashboard data, it should display the data according to the new timezone set
